packages/cli/src/environments/sourceControl/sourceControlImport.service.ee.ts

```diff
@@ -225,18 +225,13 @@ export class SourceControlImportService {
 		const ownerWorkflowRole = await this.getWorkflowOwnerRole();
 		const workflowRunner = this.activeWorkflowRunner;
 		const candidateIds = candidates.map((c) => c.id);
-		const existingWorkflows = await Container.get(WorkflowRepository).find({
-			where: {
-				id: In(candidateIds),
-			},
-			select: ['id', 'name', 'versionId', 'active'],
-		});
-		const allSharedWorkflows = await Container.get(SharedWorkflowRepository).find({
-			where: {
-				workflowId: In(candidateIds),
-			},
-			select: ['workflowId', 'roleId', 'userId'],
+		const existingWorkflows = await Container.get(WorkflowRepository).findWithFields(candidateIds, {
```

Comment:
The `WorkflowRepository` and `SharedWorkflowRepository` repositories could be injected in the constructor so there's no need to explicitly get them from the container.